scripts/monotone_framework_lattice.py

```diff
@@ -24,17 +23,20 @@ def gen_combine_values(lattice, reachability):
     out += """LatticeElement combineValues(const LatticeElement &LHS, const LatticeElement &RHS) {
 """
 
-    node_by_index = lambda index: monotone_framework.get_unique(
-        [x for x, x_data in lattice.nodes(data=True) if x_data["index"] == str(index)]
-    )
+    def node_by_index(index):
+        return monotone_framework.get_unique(
+            [x for x, x_data in lattice.nodes(data=True) if x_data["index"] == str(index)]
+        )
 
     result = defaultdict(lambda: [])
     for v1, v1_data in lattice.nodes(data=True):
         for v2, v2_data in lattice.nodes(data=True):
             if v1 != v2:
                 i1 = int(v1_data["index"])
                 i2 = int(v2_data["index"])
-                nonzero = lambda i: set([x[0] for x in enumerate(reachability[i]) if x[1] != 0])
+
+                def nonzero(i):
+                    return set([x[0] for x in enumerate(reachability[i]) if x[1] != 0])
```

Comment:
Why was that bad? It seems like pretty regular lambda. Are we banning lambda entirely?

Reply:
https://www.flake8rules.com/rules/E731.html
Lambda assignment is discouraged by pep8, inline is ok
Ok: `foo(lambda x: x+1)`
Not ok: `foo = lambda x: x+1`